src/api/rate_limiter.py

```diff
@@ -22,12 +22,22 @@ def __init__(self, request, log=None, message="", msg_args=()):
             message = f"Request %s: %s + %s {message}"
             log(message, request.method, request.url, request.cookies)
 
+class RLRequestFailed(RateLimiterError):
+    def __init__(self, request, *args, **kwargs):
+        super().__init__(request, *args, **kwargs)
```

Comment:
If I'm not mistaken the whole point of this exception is to allow future subclassing (because today only `RLRequestClientError` inherits from it) 
However it made the code a bit more obscure (especially because you `except` the parent class not this one)
So in the interest of readability maybe we could get rid of this one and instead except `RLClientError` later?